backend/sirius-web-diagrams/src/main/java/org/eclipse/sirius/web/diagrams/RectangularNodeStyle.java

```diff
@@ -35,6 +35,10 @@
 
     private int borderSize;
 
+    private int width;
+
+    private int height;
+
```

Comment:
I don't understand why this specific style should have a width and height while the node itself has this information. Given the fact that they are no JavaScript changes involved, I don't see why this file has been modified.